pkg/handlers/launch.go

```diff
@@ -214,6 +308,12 @@ func (h *launchHandler) buildEnvVars(payload *launchPayload) (map[string]string,
 }
 
 func (h *launchHandler) ServeHTTP(resp http.ResponseWriter, req *http.Request) {
+	select {
+	case <-h.availableTestRuns:
+	default:
+		http.Error(resp, "Maximum concurrent test runs reached", 429)
```

Comment:
just a micro readability nit :slightly_smiling_face: 

```suggestion
		http.Error(resp, "Maximum concurrent test runs reached", http.StatusTooManyRequests)
```